src/lib.rs

```diff
@@ -7,11 +7,15 @@
 // notice may not be copied, modified, or distributed except
 // according to those terms.
 
+#[cfg(all(feature = "runtime-tokio", feature = "runtime-async_std"))]
+compile_error!("Only one runtime can be enabled at a time");
```

Comment:
Don't forget to remove this again :)

Reply:
fixed in 85a0fed9617c36dc86f4faeadbe3a069841859ab